docs/version0.44.md

```diff
@@ -76,6 +77,16 @@ The `jcmd` tool also now supports specifying `0` as a VMID to target all VMs.
 
 For more information, see [Java diagnostic command (`jcmd`) tool](tool_jcmd.md).
 
+### Flag names that refer to the fields of `J9BuildFlags` changed in the DDR code
+
+The Direct Dump Reader (DDR) code enables reading system dump data by using the OpenJ9 Diagnostic Tool Framework for Java (DTFJ) API or the [`jdmpview`](tool_jdmpview.md) tool. The DDR code provides many built-in commands and has flag names that refer to `J9BuildFlags` to read the build flags in the system dump data.
```

Comment:
These two things are not really related.

The DDR code provides many built-in commands and has flag names that refer to `J9BuildFlags` to read the build flags in the system dump data.